packages/server-wallet/src/protocols/application.ts

```diff
@@ -29,6 +29,25 @@ const isFunded = ({app: {funding, supported}}: ProtocolState): boolean => {
   return funded;
 };
 
+// At the time of implementation, all particiapants sign turn 0 as prefund state
+// This function should also work with prefund state with increasing turn numbers.
+function myTurnToPostfund({app}: ProtocolState): boolean {
+  // I am the first participant
+  if (isPrefundSetup(app.supported) && isPrefundSetup(app.latestSignedByMe) && app.myIndex === 0) {
+    return true;
+  }
+
+  // I am NOT the first participant
+  // todo: this is not correct when there are more than 2 participants as we do not check that EVERY participant
```

Comment:
Create an issue for this?

Reply:
https://github.com/statechannels/statechannels/issues/2684